src/lib/ui/containers/ContactForm/ContactForm.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+  import { createEventDispatcher, getContext } from "svelte";
+  import { Errors, Field, faivform } from "@tuentyfaiv/svelte-form";
+  import { getTexts } from "$lib/logic/utils/objects.js";
+  import { fieldsContact } from "$lib/logic/schemas/index.js";
+
+  import type { Readable } from "svelte/store";
+  import type { FieldInputForm } from "$lib/logic/typing/globals/interfaces.js";
+  import type { ConfigForm } from "$lib/logic/typing/store/config.js";
+  import type { ContactFields, Props } from "./ContactForm.proptypes.js";
+
+  import * as stylesinternal from "./ContactForm.styles.js";
+
+  export let submit: Props["submit"];
+  export let phoneCode: Props["phoneCode"] = undefined;
```

Comment:
🚫 **[eslint]** <[no-undef-init](https://eslint.org/docs/latest/rules/no-undef-init)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>It's not necessary to initialize 'phoneCode: Props["phoneCode"]' to undefined.
```suggestion
  export let phoneCode: Props["phoneCode"];
```